### DISCUSSION TITLE
Add The Feature To Show The Channel ID In the View Channel Info (Just In Case Someone Need it)

### DISCUSSION THREAD

**Discussion by lbk2907** [BearWare/TeamTalk5#1370] [Ideas]
It will more easier just in case if someone need to get the channel ID. They can directly use the TeamTalk client to get the channel ID instead of need to open the XML file or use the custom TeamTalk client like [TTCom](https://dlee.org/teamtalk/ttcom/) from Doug Lee.

**Comment by bear101** [comment]
I don't see why the TeamTalk client should show the channel ID. The ID cannot be used for anything in the TeamTalk client, i.e. you cannot input the channel ID anywhere.

**Comment by lbk2907** [comment]
I have use 1 player created by someone. You can use the channel ID on that player to tell the player to join to the channel after log in to server. Yeah, I can use channel path but sometime it not want to join to the channel automaticly. When I try to use channel ID, it reduce the chanse for that player to stuck.
TTMediaBot: https://github.com/gumerov-amir/TTMediaBot

**Comment by tech10** [comment]
As far as I am aware, the ID of a user can't be used anywhere in the TeamTalk client either, but this is shown if you're viewing user information. Why not extend this to show a channel's ID as well, for those that may need it with custom bots or clients?

**Reply by bear101** [reply]
The user ID is used to identify users who don't have a name.

**Reply by lbk2907** [reply]
> As far as I am aware, the ID of a user can't be used anywhere in the TeamTalk client either, but this is shown if you're viewing user information. Why not extend this to show a channel's ID as well, for those that may need it with custom bots or clients?

@tech10 I agree with that.
@bear101 Maybe you can consider this.

**Comment by computertech1012** [comment]
I agree with this feature.
The channel ID is necessary in order to place bots into correct channels.
@bear101 I also think that this feature should be added, because while you can go into the XML and get the channel ID, it's not good for those who do not have access to an XML file and have no other way to get the channel ID apart from guessing.
